src/main/java/frc/robot/subsystems/climb/ClimbIOTalonFX.java

```diff
@@ -57,9 +57,9 @@ public ClimbIOTalonFX() {
                     .withKV(ClimbConstants.synced.getObject().climbkV)
                     .withKA(ClimbConstants.synced.getObject().climbkA)
                     .withKG(ClimbConstants.synced.getObject().climbkG)
-                    .withKP(ClimbConstants.synced.getObject().climbP)
-                    .withKI(ClimbConstants.synced.getObject().climbI)
-                    .withKD(ClimbConstants.synced.getObject().climbD))
+                    .withKP(ClimbConstants.synced.getObject().climbkP)
+                    .withKI(ClimbConstants.synced.getObject().climbkI)
+                    .withKD(ClimbConstants.synced.getObject().climbkD))
```

Comment:
These JSON PID constants aren't working because they aren't named correctly in the JSON files. Also maybe this is a good time to switch the names over to the format: climbKp, climbKi, climbKd, etc.